Watchman.Discord/Areas/Administration/Controllers/AdministrationController.cs

```diff
@@ -86,34 +86,24 @@ public async Task ReadUserMessages(DiscordRequest request, Contexts contexts)
         [DiscordCommand("set role")]
         public async Task SetRoleAsSafe(DiscordRequest request, Contexts contexts)
         {
-            var args = request.Arguments.Skip(1).ToArray(); // 1 args is string "role", so it's not needed
-            if (args.Length < 2)
+            var args = request.Arguments.Skip(1).ToList(); // 1 args is string "role", so it's not needed
```

Comment:
mam wrażenie że to ToList() jest wrzucone wszędzie na siłe

Reply:
nie jest na siłę, bo args jest używane 4 razy, więc uniknięcie multiple enumerate